backend/schemas/project.json

```diff
@@ -4,12 +4,12 @@
     "properties": {
         "callUrl": {
             "format": "uri",
-            "pattern": "^https://doi.org/.*$",
+            "pattern": "^https?://doi.org/.*$",
             "type": "string"
         },
         "codeUrl": {
             "format": "uri",
-            "pattern": "^https://github.com/[^/]*$",
```

Comment:
I intended to only have the github organization URL, not including any repo part.

Reply:
Ok, I changed it back.